### ISSUE TITLE
Add changelog prompt

### ISSUE THREAD

**Issue by willpwa** [Tyrrrz/DiscordChatExporter#487]
As a user, it is interesting and helpful to see the changelog when installing a new update. There is already a changelog to the releases page in the GitHub repo, but it would also be nice to have some sort of built in prompt to see the recent changes in the application.

This ticket is to investigate adding a changelog to the app - perhaps just exporting the release changelog or adding a link to it?

**Comment by Tyrrrz** [comment]
Yeah, it's a nice idea. DCE uses [Onova](https://github.com/Tyrrrz/Onova) for updates. I think it would be helpful if Onova provided a flag, something like "is this the first time running the app since last update" and in that case show a notification with "you got a new version, here's a link to the changelog".

Alternative is a popup dialog, but showing dialogs without user interaction is a big UX no-no.
